src/modules/tests/test-data/cat-a-mod2/safety-and-balance/safety-and-balance.cat-a-mod2.selector.ts

```diff
@@ -17,7 +16,7 @@ export const getSelectedBalanceQuestions = (
 
 export const safetyAndBalanceQuestionsExist = (safetyAndBalanceQuestions: SafetyAndBalanceQuestions): boolean => {
   const questions = [...safetyAndBalanceQuestions.safetyQuestions, ... safetyAndBalanceQuestions.balanceQuestions];
-  return some(questions, fault => fault.outcome != null);
+  return questions.some(fault => !!fault.outcome);
```

Comment:
Think this should be `return questions.some(fault => fault.outcome);` a double negative is equal to a positive